crates/matrix-sdk/src/widget/settings/element_call.rs

```diff
@@ -205,6 +207,31 @@ impl WidgetSettings {
     }
 }
 
+/// The Capabilities required to run a element call widget. This is intended to
+/// be used in combination with: `acquire_capabilities` of the
+/// `CapabilitiesProvider`.
+pub fn get_element_call_required_permissions() -> Capabilities {
+    use EventFilter::{MessageLike, State};
+    use MessageLikeEventFilter::WithType as MessageType;
+    use StateEventFilter::WithType as StateType;
+    Capabilities {
+        read: vec![
+            State(StateType(StateEventType::CallMember)),
+            State(StateType(StateEventType::RoomMember)),
```

Comment:
What is the room member reading capability used for, out of interest?

Reply:
We use it to fetch more detailed member data. The call.member event only contains the user id (indirectly through the state key) but we dont get: avatar + displayName.